src/components/pages/dashboard-pages/AddClass.jsx

```diff
@@ -39,17 +48,14 @@ const AddClass = () => {
           [e.target.name]: e.target.value,
         },
       });
-    } else if (e.target.name === 'date' || e.target.name === 'startTime')
+    } else if (e.target.name === 'date' || e.target.name === 'startTime') {
+      const newAvailableTime = [...formData.availableTime];
+      newAvailableTime[index][e.target.name] = e.target.value;
```

Comment:
because startTime is required, technically we can change state only even bots: date and time are filled. Now it is updated even without time 
<img width="260" alt="image" src="https://github.com/user-attachments/assets/6dffaed3-a542-4e95-b169-dac83df0aaf4">

Reply:
We can update the state for date and time separately, but  the form cannot be sent without both fields filled in.

Reply:
I know, it just allow to add new line of class time-date before you filled the previous one. I don’t think it is a big problem